src/VersionFeed.php

```diff
@@ -2,6 +2,7 @@
 
 namespace SilverStripe\VersionFeed;
 
+use SilverStripe\CMS\Model\SiteTree;
```

Comment:
Please revert the changes to the `use` statements, as they are not in any way related to the purpose of this PR

Reply:
Done.